source/time.tex

```diff
@@ -3717,8 +3717,8 @@
 
 \pnum
 \mandates
-\tcode{SourceClock::to_sys(t)} returns a \tcode{sys_time<Duration>},
-where \tcode{Duration} is a valid \tcode{chrono::duration} specialization.
+\tcode{SourceClock::to_sys(t)} returns a \tcode{sys_time<Duration2>},
+where \tcode{Duration2} is a valid \tcode{chrono::duration} specialization.
```

Comment:
This seems like a normative change to me. Before, I read this as requiring `Duration` to be a valid specialization of `chrono::duration` and requiring `SourceClock::to_sys(t)` to return `sys_time<Duration>`. The after condition is obviously quite different, allowing `SourceClock::to_sys(t)` to return a `sys_time` parameterized with a completely different specialization of `chrono::duration`.

Reply:
> I read this as [...] requiring `SourceClock::to_sys(t)` to return `sys_time<Duration>`.

That's a plausible reading on its face, but based on my experience implementing this, I think it's too restrictive to be the intended meaning. There's no guarantee that the input `Duration` will be able to represent the difference between the two clock epochs, so in general there *must* be some flexibility to return a different `duration` specialization. That's why all the built-in conversions return a `common_type_t<Duration, seconds>`. Otherwise, something reasonable like `clock_cast<gps_clock>(sys_days{1d / January / 2020y})` won't work if the input and output `Duration` have to be the same.

But I'm happy to help, if desired, if people prefer to see this escalated to an LWG issue.

Reply:
> But I'm happy to help, if desired, if people prefer to see this escalated to an LWG issue.

We've already gone to the trouble to prepare a PR, I suggest waiting for word from one of the editorial team before dropping this and resubmitting as an LWG issue. My instincts for what the Project Editor will consider to be an editorial change are not infallible.